include/nuttx/mm/mm_map.h

```diff
@@ -0,0 +1,72 @@
+/****************************************************************************
+ * include/nuttx/mm/mm_map.h
```

Comment:
mm_map.h to map.h?

Reply:
maybe a bit too generic name I'd say..

Reply:
since mm_map.h already put into include/nuttx/mm folder, it looks fine without mm_ prefix.

Reply:
ok, done